resultcapture-service/pom.xml

```diff
@@ -116,6 +108,27 @@
                 <artifactId>kotlin-maven-plugin</artifactId>
                 <groupId>org.jetbrains.kotlin</groupId>
                 <version>${kotlin.version}</version>
+                <configuration>
```

Comment:
somewhere in pom.xml we should use `<artifactId>kotlin-stdlib-jre8</artifactId>` rather than `<artifactId>kotlin-stdlib</artifactId>`